cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-deployment.adoc

```diff
@@ -0,0 +1,153 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-deployment"]
+= Tutorial: Deploying an application
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-deployment
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 23-JAN-2024
+
+== Deploying the OSToy application with Kubernetes
+
+You can deploy the OSToy application by creating and storing the images for the front-end and back-end microservice containers in an image repository. You can then create Kubernetes deployments to deploy the application. 
```

Comment:
Can we provide a link to the `OSToy application`'s official docs? for initial/additional introduction :)

Reply:
Looks good! I think it will be helpful if provided as an intro right before the actual tutorial

Reply:
Oh it already is! I forgot we had added the prereqs and overview as an assembly before this one:

https://75629--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-prerequisites